### DISCUSSION TITLE
Is it possible to run own javascript code with ChromeSafariBrowser?

### DISCUSSION THREAD

**Discussion by MarcinusX** [pichillilorenzo/flutter_inappwebview#2320] [Q&A] [answered]
I'd like to use ChromeSafariBrowser to open url but then I'd like to run my own javascript code on that page. Is that currently possible?

**Comment by pichillilorenzo** [comment, marked as answer]
Unfortunately no, that’s a native limitation of Android and iOS itself

**Reply by MarcinusX** [reply]
I see. So if I want to run custom javascript, it needs to happen via `InAppWebView` and `InAppWebViewController`, correct?

**Reply by pichillilorenzo** [reply]
Yep, correct!

**Reply by MarcinusX** [reply]
Ok, thanks for the quick response! <3
